### PR TITLE
Match, table improvements

### PR DESCRIPTION
**Pull request by blukai**

### CODE REVIEW

**Comment by blukai** [comment]
image

File: src/components/Match/matchColumns.jsx [hunk 1]
```diff
@@ -46,168 +58,148 @@ export const heroTdColumn = {
   sortFn: true,
 };
 
+const parties = (row, match) => {
+  if (match.parties && Object.values(match.parties).reduce(sum) > 0) {
```
Comment (howardchung) [inline src/components/Match/matchColumns.jsx]: what's the purpose of the second check?

Comment (blukai) [inline src/components/Match/matchColumns.jsx]: avoid situations with x10 party indicator
```
parties: {
0: 0,
1: 0,
2: 0,
3: 0,
4: 0,
128: 0,
129: 0,
130: 0,
131: 0,
132: 0
}
```

Comment (howardchung) [inline src/components/Match/matchColumns.jsx]: You might want to add initial value.  I think it's always safer to have it unless we have a reason not to.  (I think this crashes if match.parties is `{}`)

**Pull request merged**